### ISSUE TITLE
Epic: Atlas Audit3 Reconciliation

### ISSUE THREAD

**Issue by bedeho** [Joystream/atlas#2542]
## Scope
- [x] #2825
>blocked by https://github.com/Joystream/joystream/issues/3873

- [ ] #2826

Needs more details later, possibly being broken down into more issues later, with more details.

## Out of scope:
Based on https://github.com/Joystream/joystream/issues/3882
- [ ] #2824
- [ ] #2823
> blocked by QN - https://github.com/Joystream/joystream/issues/3868
> corresponding Runtime issue - https://github.com/Joystream/joystream/pull/3609 

- [ ] #2827
> Transit  is the process of being swapped: we don't need to support those features for this milestone, but worth keeping in mind.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202132419573087/1202163441442836) by [Unito](https://www.unito.io)
